src/v1.rs

```diff
@@ -308,6 +333,16 @@ impl SignatureAggregator {
         let aggregate_public_key = self.poly[0];
         let tweaked_public_key = aggregate_public_key + tweak * G;
         let c = compute::challenge(&tweaked_public_key, &R, msg);
+        let mut r_sign = Scalar::one();
+        let mut cx_sign = Scalar::one();
+        if tweak != &Scalar::zero() {
+            if !R.has_even_y() {
+                r_sign = -Scalar::one();
+            }
+            if !tweaked_public_key.has_even_y() {
+                cx_sign = -Scalar::one();
```

Comment:
ah more mathy matgic

Reply:
Yeah it really just boiled down to flipping the signs on some scalars modulo the group order.  But we had to do it in each `Signer` then reverse it in the `Aggregator`